app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlView.kt

```diff
@@ -202,6 +202,8 @@ class SessionControlView(
                 override fun onLayoutCompleted(state: RecyclerView.State?) {
                     super.onLayoutCompleted(state)
 
+                    // scroll to top placeholder
+                    view.scrollToPosition(0)
```

Comment:
I'm surprised we didn't do this earlier as part of the original fix.

Reply:
I remember we actually tried and talked about this. @rocketsroger do you recall? Was it because it didn't work without the new top placeholder which we added and then thought we don't need to scroll anymore?

Reply:
This was I also when we discovered https://github.com/mozilla-mobile/fenix/issues/22462 which would be hard to fix if we scrolled to the top on layout completion.

Reply:
Yes, we didn't add this to the original fix because on Android 10 we didn't see a need for it.  We stay on the top as long as top placeholder was populated right away.

However, on 8.1 it is definitely not the same behaviour.

Reply:
I guess another way of asking. Is the top place holder even needed if we call `scrollToPosition(0)` or would that work  without it? 

Given #22462 and if the workarounds here pile up, it might make sense to wait and address the bigger refactoring to make sure we render home in one pass from top to bottom (and from cache). However, if this makes it much better, it's OK to land in the interim.

Reply:
Unfortunately, this workaround shows the reason why we didn't go this route in the first place.  There are multiple cases that this will scroll up in the wrong time.